src_cpp/opennnBridge/bridgeController.h

```diff
@@ -0,0 +1,101 @@
+#pragma once 
+
+#include <unordered_map>
+#include <mutex>
+#include <memory>
+#include "../../opennn/opennn/opennn.h" // adding the api of opennn for creating a model
+
+#ifndef BUILDSCRIPT_PY_BRIDGECONTROLLER_H
+#define BUILDSCRIPT_PY_BRIDGECONTROLLER_H
+
+#endif //BUILDSCRIPT_PY_BRIDGECONTROLLER_H
+
+std::mutex mutex_;
+// Neural network manager singleton
+class opennnBridgeController {
+private:
+    static opennnBridgeController *instance;
+protected: // Enabling use of the clause only in other classes
+    ~opennnBridgeController() {}
+    std::unordered_map<unsigned long, std::shared_ptr<OpenNN::NeuralNetwork>> _MidNumModel; // <Mid,Model struct> Dictionary: choosing a model with  model id key
+                                                                                  // SANN::Model to OpenNN::NeuralNetwork
+    opennnBridgeController(){}
+
+public:
+    /**
+     * Singletons should not be cloneable.
+     */
+    opennnBridgeController(opennnBridgeController &other) = delete; //If another instance is crated , delet it (& = adress)
+    /**
+     * Singletons should not be assignable.
+     */
+    void operator=(const opennnBridgeController &) = delete; //disables creating a reference of x in y (& after var = reference)
+
+    // TODO: Think about locking mechanism
```

Comment:
remove this old documentation 
We already know what the locking mechanism is

Reply:
done